lib/dialect/src/Passes.td

```diff
@@ -1,6 +1,11 @@
 include "mlir/Pass/PassBase.td"
 include "mlir/Rewrite/PassUtil.td"
 
+def ConstraintsPass : Pass<"rlc-constraints", "mlir::ModuleOp"> {
```

Comment:
Rename in rlc-test-function-constrainsts, the analysis itself will be run by whoever needs it, the pass is here so that we can test the analysis in isolation.